request_handlers.cc

```diff
@@ -48,10 +51,62 @@ void StaticRequestHandler::handleRequest(const char* request_buffer,
     std::cout << t << "." << std::endl;
   }
 
+  int i = 0;
+  auto curToken = tokens.begin();
+  std::string resourcePath;
+  for (;;) {
+    if (curToken == tokens.end()) {
+      // TODO: tokens is too short
+      std::cout << "DEBUG: tokens in RequestRouter is too short" << std::endl;
+      return;
+    }
+
+    if (i == 1) {
+      resourcePath = *curToken;
+    }
+
+    i++;
+    curToken++;
+  }
+
   // TODO: Check integrity of path
-  // TODO: Determine file extension (for later, to fill out Content-Type)
+  std::string request_path = resourcePath;
+  if (request_path.empty() || request_path[0] != '/'
+      || request_path.find("..") != std::string::npos)
+  {
+    // TODO: Indicate bad request
```

Comment:
Can we add a temporary `"DEBUG"` print-out here so that we're aware of this bad request during testing?